### ISSUE TITLE
governance strategy - member dashboard proposal feedback

### ISSUE THREAD

**Issue by patrick727** [rchain/bounties#400]

**Comment by 9rb** [comment]
[Governance overview proposal - Short / Medium / Long Term](https://docs.google.com/document/d/1ubAJ1Wa1N-incLNtk2dGSznDoJfuc82F2n35u4kanv4/edit?ts=5a89d94e)  and [Short-term: members portal services](https://docs.google.com/document/d/1JXM8k-fzf78Y_GbfkDdsekmdeVqqifC4VM0zEMBDxDw/edit) are relevant to this discussion.

**Comment by 9rb** [comment]
I made some edits to those docs, especially the dashboard one ..

**Comment by dckc** [comment]
@patrick727 please change "governance" to "onboarding" (or "welcoming new members") in the issue title and document title. Let's focus. Governance is a big ball of wax. Onboarding is something we can and should address in a tractable amount of time (not completely, but we should get to an 80% point, to where this is not the biggest issue).

And I'll repeat: I strongly encourage each person to personally welcome one or more other members. (by way of example: #295, #294, #293; for efforts by others, see #15 and #253).

**Comment by patrick727** [comment]
nathan windsor referenced https://whycardano.com/ as a good wiki like info source

**Comment by patrick727** [comment]
(dont want to put words in mouths)

but its my understanding that ian, jeremy and dan are leaning towards full wordpress installation for member site for many reasons, one being you can build and integrate php apps such as discord authentication and bounty web app. 

Thoughts

**Comment by dckc** [comment]
I don't doubt that the wordpress install will serve more purposes later. But I appreciate the initial focus.

**Comment by patrick727** [comment]
i agree, so right now we have vague consensus around utilizing wordpress for short/mid term member solutions

Ok nice that's a step forward

**Comment by patrick727** [comment]
Hey All, thanks for all of the feedback, in two days I will close this issue down and then create another one for "mock up of membership website" based on these requirements for myself and @nathanwindsor to execute on. 

Get your feedback in, also if you are good at web mockups(@pmoorman , @AyAyRon-P) reach out

**Comment by AyAyRon-P** [comment]
@patrick727 I can develop mock up options once the parameters have been specified based on the feedback over the next two days. Over the next two days it would be good if we could determine a list of MVP features to integrate into the dashboard to speed up the execution of the project.

**Comment by kitblake** [comment]
I see goals but is there a structure for the Members site? Would be good to have it before beginning mockups. We could make an outline with indents in the GDoc for an initial setup.

**Comment by AyAyRon-P** [comment]
@kitblake I think an outline would be equally as crucial as the MVP list for the sake of navigation. I'm free today if you want to go over a preliminary outline.

**Comment by pmoorman** [comment]
@patrick727 I would really like to be involved with a WG focused on onboarding. I guess what I wrote in #253 more or less outlines my perspective on things.

On a practical note: I haven't got time now to review things today or tomorrow (because I'm on the road). Sorry about that.

**Comment by dckc** [comment]
@pmoorman there was a meeting yesterday evening and I gather* the outcome is that one group is coalescing around new member orientation and a separate one around the web site. @patrick727 is leading the web site group. So I plan to split the **comms-onboarding** label into **new-member** and **website** or the like.

*_grumble grumble_ I should be able to point you to the notes, but they're not shared yet. _grumble grumble_

**Comment by pmoorman** [comment]
@dckc thanks. I'm talking with Patrick about the branding/website WG (main website, not members area), and with @makys about the onboarding stuff.

But if I get you right, there are actually 2 onboarding WGs, right? One is the "greeters" and one is the "members portal", which can somewhat overlap. 

Does anyone know _**why**_ it was decided to break the onboarding into 2 groups, rather than cluster it together?

**Comment by pmoorman** [comment]
@patrick727 @kitblake @AyAyRon-P I went through the "short term: members site" document, and I think the indenting to arrive at a basic architecture is a really good step forward.

### Maybe I can make a few suggestions (to consider):

1. **Process  ::**  We probably need to start thinking about the approval/governance process. What's needed to get budget, and get a free pathway to actually execute this?

2. **Problem definition  ::**  I would suggest — preceding the listing of features — that we outline which specific problems we're actually aiming to solve for members. It might be that certain solutions/features can be replaced by other features that would for instance be easier.

3. **Add Email  ::**  It might be worth connecting this to the #219 discussion about email marketing providers. I suspect that having email integrated with the members portal will make many features much cheaper/easier/faster to implement.  _(depends on exact requirements though!)_

4. **Education  ::**  In terms of the onboarding of people, I think we should really add some educational base, too. This might cluster with "visualisation", which I think is actually a sub-group of education (or a particular way to go about education)

Other than that, I'm happy to see all the energy as of late surrounding onboarding, and I think we're really moving in the right direction with this effort.  

I would really like to be involved further with this!! I could help with either spec'ing out the scope, or also with the more practical parts of implementation.

The biggest hurdle I foresee at the moment, is how we get this all approved and moving nicely from a governance perspective. Maybe @kitblake can help navigate there, though (through exec committee)

**Comment by patrick727** [comment]
Great stuff guys! Im going to close this issue down now and create the new one. 

@AyAyRon-P @pmoorman @kitblake id love for you to take part in the working group. 

MVP features should be in the document

Onboarding
Voting
Member comms
Petition system

and then..
visualization
news

**Comment by dckc** [comment]
"Onboarding" isn't clear enough as a feature, to me. I suggest refining it with one or more user stories.

**Comment by dckc** [comment]
And "Member comms" is no smaller than "member web site"

**Comment by dckc** [comment]
@pmoorman writes:
> Does anyone know **why** it was decided to break the onboarding into 2 groups, rather than cluster it together?

Not really; my guess is: leaders stepped up for each of the parts. That works for me: _s/he who does the work makes the rules._

**Comment by dckc** [comment]
oh... also... in today's executive committee meeting, @kitblake said:
> There’s a lot of momentum in the Member portal WG. ... (There is some overlap with the Greeters WG, but I’m in both and can serve as a linker).

**Comment by pmoorman** [comment]
yeah, oke. I spoke with @makys this morning about the same thing.

For reference for anyone else who's interested, the breakdown between WGs is basically as follows:

- Makys / "greeters" WG is about the **human part** of the onboarding equation: giving Ian a breather in the KYC process, and also just really greeting people and helping them get started. I think they should oversee intangible stuff like "purpose and principles" (#371) also. 

- Patrick / "members portal" group is about all the **automation part.** They'll also tackle other things that aren't necessarily onboarding, but are part of the portal regardless. 

Looks like I'll actually also be in both, so I guess the overlap is not something much to worry about then, if Kit and I both serve double duties.
